composer.json

```diff
@@ -13,6 +13,7 @@
     "require-dev": {
         "behat/behat": "^3.10",
         "composer/installers": "~1.5",
+        "composer/xdebug-handler": "^2.0",
```

Comment:
Please remove this as it has been fixed in code-review: https://github.com/openeuropa/code-review/blob/2.x/composer.json#L7